server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -1014,8 +1013,7 @@ protected Node(
                 searchModule.getQueryPhase(),
                 searchModule.getFetchPhase(),
                 responseCollectorService,
-                circuitBreakerService,
-                searchModule.getIndexSearcherExecutor(threadPool)
+                circuitBreakerService
```

Comment:
Please revert this change, the `IndexSearcherExecutor` could be provided by the plugins since we opened this extension point.